packages/server/socketHelpers/sendGQLMessage.ts

```diff
@@ -6,19 +6,18 @@ interface Message {
   type: string
   syn: boolean
   payload?: Record<string, any>
-  id?: string
+  id: string
 }
 
 const sendGQLMessage = (
   context: ConnectionContext,
+  opId: string,
```

Comment:
now that opId is guaranteed in all responses, we can move it up the parameter list